force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -190,9 +308,17 @@ public with sharing class FlowControlLoopsCollections {
         // add 3
 
         // Return the populated list
-        return null; // Replace null with the variable you used to store the result
+
+        List<Integer> myIntegerList = new List<Integer>();
+
+        myIntegerList.add (1);
+        myIntegerList.add (2);
+        myIntegerList.add (3);
+
+        return myIntegerList; // Replace null with the variable you used to store the result
```

Comment:
Good job. A tip is that if you know the values of your collection at write time you can pass them into the constructor.  This is a little faster. 

```
  public static List<Integer> createAndPopulateList() {
      return new List<Integer>{1, 2, 3};
    }
 ```